CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.3.0] - 2024-04-11
+
+### Changed
+- Updated the enum class [AgentFeedbackState] to include the new state [AgentFeedbackState.assigned] and replaced [AgentFeedbackState.inProgress] with [AgentFeedbackState.started] (#120)
```

Comment:
This has to be a breaking change. You must prefix this change log with \*\*BREAKing\*\*